website/package.json

```diff
@@ -14,6 +15,7 @@
     "react": "~0.13.0",
     "react-docgen": "^2.0.1",
     "react-page-middleware": "git://github.com/facebook/react-page-middleware.git",
-    "request": "latest"
+    "request": "latest",
```

Comment:
would be great to send a follow up to use correct versions instead of "latest". Otherwise it's going to break in the future and going to be a pain to figure out why